Controller/RacetimeChannel.cs

```diff
@@ -315,6 +323,8 @@ private void UpdateRaceData(RaceMessage msg)
             if (msg.Race != null)
                 Race = msg.Race;
 
+            UpdateRaceComparisons(Race);

```

Comment:
As msg.Race can be null in cases and we're only setting race in those chases, we should probably move this within the if statement